### ISSUE TITLE
MO2 Alpha 2 - Massive LOG File(s) Generated

### ISSUE THREAD

**Issue by NeoTU** [TanninOne/modorganizer#407]
Noticed my HDD space chewed up and discovered it was MO2.
Can log generation be turned off so this stops happening? 
Not the first, once there was a 200GB and a 80GB one there that I deleted. Glad I'm not on a SSD...

![mo2logs](https://cloud.githubusercontent.com/assets/11801357/11499072/41097c2c-986f-11e5-9535-24f1a11d3aa0.jpg)

**Comment by NoShotz** [comment]
damn, thats pretty big, biggest ive seen is 200mb

**Comment by NeoTU** [comment]
I changed the logs folder to be Read Only to temporary resolve this issue. Seems to be working.

**Comment by grasmann** [comment]
Holy shit.
You can change the log level in the settings.
It will probably log much less if you change it to error unless the files basically are full of errors.
Biggest one I've seen on my pc was ~5mb.

**Comment by WhiteZeroX** [comment]
Change your log level down from Debug perhaps?

**Comment by OrdinaryMagician** [comment]
Yeah, it defaults to Debug, likely because you know, it's an alpha.

**Comment by UlithiumDragon** [comment]
I am on an SSD, and I only noticed AFTER my disc yelled at me for running out of space /cry
